pkg/scheduler/apis/config/validation/validation_pluginargs_test.go

```diff
@@ -114,7 +114,7 @@ func TestValidateDefaultPreemptionArgs(t *testing.T) {
 		t.Run(name, func(t *testing.T) {
 			err := ValidateDefaultPreemptionArgs(tc.args)
 			if diff := cmp.Diff(tc.wantErrs.ToAggregate(), err, ignoreBadValueDetail); diff != "" {
-				t.Fatalf("ValidateDefaultPreemptionArgs returned err (-want,+got):\n%s", diff)
+				t.Errorf("ValidateDefaultPreemptionArgs returned err (-want,+got):\n%s", diff)
```

Comment:
`Errorf` will log the error and continue IIUC, what's the point of this change?

Reply:
Semantically this failure doesn't mean the test case should exit immediately (although there are no more things to do here).

Reply:
I cannot see any connection between this and what's the PR want to address, leave it open to other reviewers.

Reply:
This change doesn't related to this PR indeed. It fixes mistakes in my previous PR #96745